inference-engine/src/mkldnn_plugin/nodes/psroi.cpp

```diff
@@ -46,22 +50,36 @@ class PSROIPoolingImpl: public ExtLayerBase {
             part_size_ = layer->GetParamAsInt("part_size", 1);
             trans_std_ = layer->GetParamAsFloat("trans_std", 1);
 
+            Precision inputPrecision = layer->insData[0].lock()->getTensorDesc().getPrecision();
+
             if (no_trans_) {
-                addConfig(layer, {DataConfigurator(ConfLayout::PLN), DataConfigurator(ConfLayout::PLN)}, {DataConfigurator(ConfLayout::PLN)});
+                addConfig(layer, {DataConfigurator(ConfLayout::PLN, inputPrecision), DataConfigurator(ConfLayout::PLN, Precision::FP32)},
+                                 {DataConfigurator(ConfLayout::PLN, Precision::FP32)});
+
+                addConfig(layer, {DataConfigurator(ConfLayout::BLK16, inputPrecision), DataConfigurator(ConfLayout::PLN, Precision::FP32)},
```

Comment:
Why we don't support BLK8 layout? We will still have reorders on legacy platforms (SSE42, AVX2)

Reply:
Will be done.